org.dpsoftware.FireflyLuciferin.json

```diff
@@ -0,0 +1,100 @@
+{
+  "id": "org.dpsoftware.FireflyLuciferin",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
```

Comment:
why not use 24.08?

Reply:
reading the documentation I tought that 23.08 was the latest stable. updated and tested it. thanks